packages/circom/tests/circuits/simple_regex_decomposed.json

```diff
@@ -0,0 +1,16 @@
+{
+  "parts":[
+    {
+      "is_public": false,
+      "regex_def": "email was meant for @"
+    },
+    {
+      "is_public": true,
+      "regex_def": "[a-zA-Z]+"
```

Comment:
Huh this one should also have numbers and underscores, it's the twitter username one